test/mobile/custom_build/build.sh

```diff
@@ -0,0 +1,145 @@
+#!/bin/bash
+###############################################################################
+# This script shows/tests the flow to build libtorch locally with optimized
```

Comment:
Mentioning a pet peeve of mine: I generally don't like build scripts intended for general use by users, because they can make it difficult to toggle build parameters on the underlying cmake invocations (we had this problem for the longest time in setup.py until xuhdev wrote a general framework for passing environment variables into cmake flags for the underlying cmake invocation).

Reply:
removing "shows" from the comment to avoid confusing external developers :)

Reply:
Not too sure xuhdev's stuff in setup.py is all that useful, since it's specific for building PyTorch on Windows. But worth taking a look :)